Add --port-file command line option to JSON API (#5454)
digital-asset/daml: docs/source/json-api/index.rst

      --address <value>
            IP address that HTTP JSON API service listens on. Defaults to 127.0.0.1.
      --http-port <value>

>             HTTP JSON API service port number. A port number of 0 will let the system pick up an ephemeral port. Consider specifying port-file option with port number 0.

Reviewer: ```suggestion
            HTTP JSON API service port number. A port number of 0 will let the system pick an ephemeral port. Consider specifying the ``--port-file`` option with port number 0.
```

      --port-file <value>

>             Optional file where to write the allocated HTTP port number. Used to inform clients in CI about which port HTTP JSON API listens on. Defaults to none, that is file does not get created/updated.

Reviewer: ```suggestion
            Optional file where to write the allocated HTTP port number. Used to inform clients in CI about which port HTTP JSON API listens on. Defaults to none, that is, no file gets created/updated.
```

>             Optional file where to write the allocated HTTP port number. Used to inform clients in CI about which port HTTP JSON API listens on. Defaults to none, that is file does not get created/updated.

Reviewer: ```suggestion
            Optional file where to write the allocated HTTP port number. This can be used to inform clients in CI about which port the HTTP JSON API listens on. Defaults to none, that is, no file gets created/updated.
```

      --address <value>
            IP address that HTTP JSON API service listens on. Defaults to 127.0.0.1.
      --http-port <value>

>             HTTP JSON API service port number. A port number of 0 will let the system pick an ephemeral port. Consider specifying `--port-file` option with port number 0.

Reviewer: Isn't it like this in RST?
```suggestion
            HTTP JSON API service port number. A port number of 0 will let the system pick an ephemeral port. Consider specifying ``--port-file`` option with port number 0.
```

Author: yes, it is like this in rst, but this is a copy/paste from `daml json-api --help`.
